cli/cmd/testdata/install_single_namespace_output.golden

```diff
@@ -804,7 +820,13 @@ spec:
             add:
             - NET_ADMIN
           privileged: false
+          runAsGroup: 0
+          runAsNonRoot: false
+          runAsUser: 0
         terminationMessagePolicy: FallbackToLogsOnError
+      securityContext:
+        runAsGroup: 472
+        runAsUser: 472
```

Comment:
Since these IDs are container-specific, my preference is to set them on the container itself, rather than on the pod. Can I trouble you to update this PR so that all of the new security contexts are set per-container instead of per-pod?